### PR TITLE
Refactor models module

### PR DESCRIPTION
**Pull request by dhruvtapasvi**

### CODE REVIEW

File: src/model/loss/meanSquaredErrorLoss.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import numpy as np
+from keras.metrics import mean_squared_error
+from keras.backend import mean, flatten
+
+
+def meanSquaredErrorLossConstructor(inputRepresentationDimensions):
+    totalNumberOfPixels = np.prod(inputRepresentationDimensions)
+
+    def meanSquaredErrorLoss(inputRepresentation, decodedInputRepresentation):
+        return mean(totalNumberOfPixels * mean_squared_error(flatten(inputRepresentation, flatten(decodedInputRepresentation))))
```
Comment (dhruvtapasvi) [inline src/model/loss/meanSquaredErrorLoss.py]: Fix bracketing on this line
